### PR TITLE
Remove support for list item with *

### PR DESCRIPTION
**Pull request by p-salmon**
With the latest version of Quill, the support for list starting with `* ` has been added by default:
https://github.com/quilljs/quill/pull/1819

The following line were conflicting with this behaviour, creating weird behaviour where two lines were being cleaned.

We've removed the support for `* ` but let the support for `+ `.
We haven't renamed the match and let the `asterisk-list` intentionally, happy to change it if needed.

Thanks again for the awesome project, it's been really useful for us!

### CODE REVIEW

**Comment by patleeman** [comment]
Thanks for updating this.

I'm thinking we should update the minor version to reflect this fairly large change. What do you think?

Also, do you think we should pin the dependency in the package.json to the latest version?

File: src/index.js [hunk 1]
```diff
@@ -186,7 +186,7 @@ class MarkdownShortcuts {
       },
       {
         name: 'asterisk-ul',
-        pattern: /^(\*|\+)\s$/g,
+        pattern: /^\+\s$/g,
```
Comment (patleeman) [inline src/index.js]: Can you add a note somewhere here that the asterisk was removed in quill version xx.xx.xx?

Thanks.

**Comment by p-salmon** [comment]
@patleeman Thanks for your review, I've added a comment and updated the version. I can squash the commit for another PR if needed.

> Also, do you think we should pin the dependency in the package.json to the latest version?

That's up to you

File: src/index.js (hunk 1 above)
Comment (p-salmon) [reply src/index.js]: I added a comment the line above.

**Comment by p-salmon** [comment]
Hey @patleeman ! Any news on this?

**Comment by patleeman** [comment]
Thanks for the work on this @p-salmon

**Comment by p-salmon** [comment]
Hey @patleeman ! It's been a while but would be nice to have a new version with this fix ^^ Thanks!

**Pull request merged**